Resources/doc/index.rst

```diff
@@ -24,21 +28,45 @@ Getting Started With FOSRestBundle
 Installation
 ------------
 
-Installation is a quick (I promise!) one-step process:
+Before using this bundle in your project, add it to your composer.json file:
+
+.. code-block:: bash
+
+    $ composer require friendsofsymfony/rest-bundle
+
+Then, like for any other bundle, include it in your Kernel class::
+
+    public function registerBundles()
+    {
+        $bundles = array(
+            // ...
+
+            new FOS\RestBundle\FOSRestBundle(),
+        );
+
+        // ...
+    }
+
+At last, make sure you have a serializer enabled. The bundle will look at these
+three possibilities **in this order** to decide which serializer to use:
+
+#. The service ``fos_rest.service.serializer`` if it is configured (must be an
+instance of ``FOS\RestBundle\Serializer``).
+#. The JMS serializer, if the `JMSSerializerBundle`_ is available (and registered).
+#. The `Symfony Serializer`_
 
-1. :doc:`Setting up the bundle <1-setting_up_the_bundle>`
+.. _`JMSSerializerBundle`: https://github.com/schmittjoh/JMSSerializerBundle
```

Comment:
Maybe link to the bundle's documentation (http://jmsyst.com/bundles/JMSSerializerBundle) instead?